src/qibolab/platforms/abstract.py

```diff
@@ -64,6 +65,9 @@ class Qubit:
     mixer_readout_g: float = 0.0
     mixer_readout_phi: float = 0.0
 
+    # Ask Stavros if it is iq_angle
+    rotation_angle: float = 0.0
+
```

Comment:
Yes, this is `iq_angle`. `rotation_angle is now deprecated, it can be removed.

Reply:
I removed from all the runcards, but then, there was a test failing because QM code: 

`    def register_integration_weights(self, qubit, readout_len):
        """Registers integration weights in QM config.

        Args:
            qubit (:class:`qibolab.platforms.quantum_machines.Qubit`): Qubit
                object that the integration weights will be used for.
            readout_len (int): Duration of the readout pulse in ns.
        """
        iq_angle = qubit.iq_angle
        self.config["integration_weights"].update(
            {
                f"cosine_weights{qubit.name}": {
                    "cosine": [(np.cos(iq_angle), readout_len)],
                    "sine": [(-np.sin(iq_angle), readout_len)],
                },
                f"sine_weights{qubit.name}": {
                    "cosine": [(np.sin(iq_angle), readout_len)],
                    "sine": [(np.cos(iq_angle), readout_len)],
                },
                f"minus_sine_weights{qubit.name}": {
                    "cosine": [(-np.sin(iq_angle), readout_len)],
                    "sine": [(-np.cos(iq_angle), readout_len)],
                },
            }
        )`
        
That is why re-introduced the attribute in the Qubit. I wanted to ask @stavros11 if I could modify the code,but I've done it and I am now checking if the code passed the tests.